jfr-agent-extension/build.gradle.kts

```diff
@@ -27,12 +27,16 @@ tasks.shadowJar {
                 "Implementation-Vendor" to "New Relic, Inc."
         )
     }
+    // Ensure module-info.class files from dependencies don't erroneously make it into the jar
+    exclude("**/module-info.class")
+    exclude("module-info.class")
```

Comment:
This would only be useful if someone tried to place a dependency on the agent extension, no?

Reply:
No, this is actually required to stop the build choking.